src/components/Form/Calculator/Results/index.js

```diff
@@ -0,0 +1,29 @@
+const Results = ({toPayPerMonth, payOffTimeMonths, payOffTimeWholeYears, payOffTimeExtraMonths, payOffYearly}) => {
+
+    console.log(payOffTimeWholeYears)
+    console.log('hello')
+    console.log(payOffYearly)
```

Comment:
A note:

If you want to console.log() props like this, especially those that are going to change (ie things coming from state somewhere) your best bet is to do it in a useEffect() where the second parameter contains the things you wish to log.

```
useEffect(() => {
    console.log(payOffTimeWholeYears)
    console.log(payOffYearly)
},[payOffTimeWholeYears, payOffYearly])
```

That way you will always see the correct current value of those variables (remember lots of react is async, meaning the console.log could run before a new piece of state gets passed in, even if it's after it in the code. A useEffect setup as above will guarantee that the console.logs run every time the data changes.